scripts/flash_usb.py

```diff
@@ -340,13 +340,22 @@ def flash_rp2040(options, binfile):
         sys.exit(-1)
 
 MCUTYPES = {
-    'sam3': flash_atsam3, 'sam4': flash_atsam4, 'same70': flash_atsam4,
-    'samd': flash_atsamd, 'same5': flash_atsamd,
-    'lpc176': flash_lpc176x, 'stm32f103': flash_stm32f1,
-    'stm32f4': flash_stm32f4, 'stm32f042': flash_stm32f4,
-    'stm32f072': flash_stm32f4, 'stm32g0b1': flash_stm32f4,
-    'stm32h7': flash_stm32f4, 'stm32l4': flash_stm32f4,
-    'stm32g4': flash_stm32f4, 'rp2040': flash_rp2040,
+    'lpc176': flash_lpc176x,
+    'rp2040': flash_rp2040,
+    'sam3': flash_atsam3,
+    'sam4': flash_atsam4,
+    'samd': flash_atsamd,
+    'same5': flash_atsamd,
+    'same70': flash_atsam4,
+    'stm32f042': flash_stm32f4,
+    'stm32f072': flash_stm32f4,
+    'stm32f103': flash_stm32f1,
+    'stm32f4': flash_stm32f4,
+    'stm32f7': flash_stm32f4,
+    'stm32g0b1': flash_stm32f4,
+    'stm32g4': flash_stm32f4,
+    'stm32h7': flash_stm32f4,
+    'stm32l4': flash_stm32f4,
```

Comment:
Please don't mix whitespace changes with functional changes.  It makes review harder and complicates merging and use of tools like `git revert`.